src/three/TilesRenderer.js

```diff
@@ -580,13 +580,22 @@ export class TilesRenderer extends TilesRendererBase {
 		const loadIndex = tile._loadIndex;
 		let promise = null;
 
+		const m = new Matrix4();
+		m.set(1.0, 0.0,  0.0, 0.0,
+			0.0, 0.0, -1.0, 0.0,
+			0.0, 1.0,  0.0, 0.0,
+			0.0, 0.0,  0.0, 1.0);
+
```

Comment:
Instead of using a fixed, explicit matrix we should use the same matrix that was being applied previously from here:

https://github.com/NASA-AMMOS/3DTilesRendererJS/blob/ff23126bf6cbcc3faad9a9e2eedde0d6beaadf57/src/three/TilesRenderer.js#L663-L677